inlineplz/interfaces/github.py

```diff
@@ -20,6 +20,7 @@ def __init__(self, owner, repo, pr, token, url=None):
             self.github = github3.GitHub(token=token)
         else:
             self.github = github3.GitHubEnterprise(url, token=token)
+        self.user = self.github.me()
```

Comment:
Yeah that's probably much more sane. Prefix inline-plz comments, search for comments with that prefix.